cmd/juju/application/resumerelation_test.go

```diff
@@ -68,32 +67,32 @@ func (s *ResumeRelationSuite) TestResumeRelationFail(c *gc.C) {
 	s.mockAPI.SetErrors(errors.New(msg))
 	err := s.runResumeRelation(c, "123")
 	c.Assert(err, gc.ErrorMatches, msg)
-	s.mockAPI.CheckCall(c, 0, "SetRelationStatus", 123, relation.Joined)
+	s.mockAPI.CheckCall(c, 0, "SetRelationsStatus", 123, false)
 	s.mockAPI.CheckCall(c, 1, "Close")
 }
 
 func (s *ResumeRelationSuite) TestResumeRelationBlocked(c *gc.C) {
 	s.mockAPI.SetErrors(common.OperationBlockedError("TestResumeRelationBlocked"))
 	err := s.runResumeRelation(c, "123")
 	coretesting.AssertOperationWasBlocked(c, err, ".*TestResumeRelationBlocked.*")
-	s.mockAPI.CheckCall(c, 0, "SetRelationStatus", 123, relation.Joined)
+	s.mockAPI.CheckCall(c, 0, "SetRelationsStatus", 123, false)
 	s.mockAPI.CheckCall(c, 1, "Close")
 }
 
 type mockResumeAPI struct {
 	*testing.Stub
 	version            int
-	resumeRelationFunc func(relationId int, status relation.Status) error
+	resumeRelationFunc func(relationId int, suspended bool) error
```

Comment:
odd name? should be setRelationSuspendedFunc?

Reply:
ok